style.css

```diff
@@ -82,3 +82,42 @@ a:active {
 .wp-block-button__link:hover {
 	opacity: 0.90;
 } 
+
+/*
+ * Alignment styles.
+ * These rules are temporary, and should not be relied on or 
+ * modified too heavily by themes or plugins that build on 
+ * Twenty Twenty-Two. These are meant to be a precursor to 
+ * a global solution provided by the Block Editor. 
+ * 
+ * Relevant issues:
+ * https://github.com/WordPress/gutenberg/issues/35607
+ * https://github.com/WordPress/gutenberg/issues/35884
+ */
+
+.wp-site-blocks,
+.is-root-container,
```

Comment:
In the site editor, check out the archive or home template. This rule is applying an unwanted padding in the query: 

<img width="1904" alt="Screen Shot 2021-12-14 at 10 29 03 AM" src="https://user-images.githubusercontent.com/5375500/146028580-d639e881-6e99-4cd3-93be-37efb810c768.png">

Reply:
Yeah, I believe `is-root-container` is used in multiple places. We should switch to something else.

Reply:
I just pushed a potential fix in https://github.com/WordPress/twentytwentytwo/pull/291/commits/c4a80d3b90992f276b66284aab2e6429575b0dc7. It uses more specific selectors, but also required specifying both the root container for the site editor and for the post/page editor.

Reply:
Hmm actually, now that I'm looking closer I'm not sure that'll do it. 😕

Reply:
😕 It doesn't. Reverting.

Reply:
Ok, after some further digging, I can see that this is basically a re-creation of this issue: https://github.com/WordPress/gutenberg/issues/33248. 

Basically, the Query block renders using some nested DOM elements, and that causes this duplicate selector where it shouldn't be. In the post/page editor, Gutenberg itself provides some `is-root-container` styles that cause the same behavior you're seeing here. 

Since https://github.com/WordPress/gutenberg/issues/33248 is listed as a must-have for 5.9, and is currently in progress I think we're ok to push through with this bug present for now.